jobs/integr8ly/installation-pipeline-qe-pony.yaml

```diff
@@ -0,0 +1,93 @@
+---
+
+- job:
+    name: installation-pipeline-qe-pony
+    project-type: pipeline
+    description: "Installs Integreatly remotely from Jenkins slave to QE Pony cluster."
+    sandbox: false
+    concurrent: true
+    parameters:
+        - string:
+            name: REPOSITORY
+            default: https://github.com/integr8ly/installation.git
+            description: "Repository of the Integreatly installer"
+        - string:
+            name: BRANCH
+            default: 'master'
+            description: "Branch of the installer repository"
+        - string:
+            name: CLUSTER_URL
+            description: "URL of cluster for installation."
+        - string:
+            name: GH_CLIENT_ID
+            description: "GitHub Client ID for OAuth Apps, required for some of the walkthroughs. Can be left empty"
+        - string:
+            name: GH_CLIENT_SECRET
+            description: "GitHub Client Secret for OAuth Apps, required for some of the walkthroughs. Can be left empty"
+        - bool:
+            name: SELF_SIGNED_CERTS
+            default: false
+            description: "Indicates whether cluster uses self signed certificates or not"
+        - string:
+            name: ANSIBLE_USER
+            default: root
+            description: "User for Ansible to access the master node of target cluster"
+        - string:
+            name: MASTER_URLS
+            description: "Comma separated list of URLs for master nodes of target cluster to be used in Ansible inventory file"
+        - string:
+            name: OC_USER
+            description: "OpenShift user for QE cluster."
+        - string:
+            name: OC_PASSWORD
+            description: "OpenShift password for QE cluster."
+    dsl: |
+        timeout(60) {
+            node('cirhos_rhel7') {        
+                stage('Verify input') {
+                    if (!MASTER_URLS) {
```

Comment:
Why there is only MASTER_URLS missing check, if there are some other required parameters. Maybe we should check for those as well if we wanna do it for MASTER_URLS

Reply:
Well yes it would make sense to check CLUSTER URL and USER credentials.